### ISSUE TITLE
Run-by-line for native notebooks

### ISSUE THREAD

**Issue by greazer** [microsoft/vscode-jupyter#255]

**Comment by DonJayamanne** [comment]
Blocked on upstream https://github.com/microsoft/vscode/issues/105183

**Comment by DavidKutu** [comment]
closing this in favor of https://github.com/microsoft/vscode-jupyter/issues/5607
